src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,31 @@
+.stars {
+  display: flex;
+  column-gap: 4px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--0 .stars__star {
+  background-image: url(../images/star.svg);
+}
```

Comment:
This block of code seems to be unnecessary. You are setting the background image to `star.svg` for the stars in a `stars--0` block, but this is already done in the `stars__star` class. This redundancy can lead to confusion and should be avoided.